modules/rules_libusb/metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+    "homepage": "https://pigweed.googlesource.com/pigweed/rules_libusb/",
+    "maintainers": [
+        {
+            "github": "armandomontanez",
+            "name": "Armando Montanez"
+        },
+        {
+            "github": "tpudlik",
+            "name": "Ted Pudlik"
+        },
+        {
+            "github": "cramertj",
+            "name": "Taylor Cramer"
+        },
+        {
+            "email": "pigweed-team@googlegroups.com",
+            "name": "The Pigweed Team"
+        }
+    ],
+    "repository": [
+        "https://pigweed.googlesource.com/pigweed/rules_libusb/"
```

Comment:
you can just remove this line if your source archives aren't going to come from here.